package.json

```diff
@@ -12,7 +12,7 @@
   "repository": "https://github.com/atom/one-dark-ui",
   "main": "lib/main",
   "engines": {
-    "atom": ">0.40.0"
+    "atom": ">1.0.11"
```

Comment:
I think you're right. Should try the semver module to make sure: https://github.com/npm/node-semver